news/newsflash.txt

```diff
@@ -4,6 +4,9 @@
 # Format is two fields, colon-separated; the first line is the column
 # headings.  URL paths must all be absolute.
 Date: Item
+26-Feb-2019: OpenSSL 1.1.1c is now available, including bug and security fixes
+26-Feb-2019: OpenSSL 1.1.0k is now available, including bug and security fixes
+26-Feb-2019: OpenSSL 1.0.2s is now available, including bug fixes
```

Comment:
28-May-2019 !!

Reply:
One at a time, please :wink:

Reply:
I learned that with you I need to be very fast, Monsieur Le Vite!  ;-)